templates/deno/docs/example.md.twig

```diff
@@ -16,10 +16,17 @@ client
     .set{{header}}('{{node[header]['x-appwrite']['demo']}}') // {{node[header].description}}
 {% endfor %}
 {% endfor %};
-
 {% endif %}
 
-let promise = {{ service.name | caseCamel }}.{{ method.name | caseCamel }}({% for parameter in method.parameters.all %}{% if parameter.required %}{% if not loop.first %}, {% endif %}{{ parameter | paramExample }}{% endif %}{% endfor %});
+const promise = {{ service.name | caseCamel }}.{{ method.name | caseCamel }}({% for parameter in method.parameters.all %}
+{% if loop.first %}
+
+    {% endif %}{% if parameter.required %}
```

Comment:
whats the reason for this empty if condition?

Reply:
Oh, if no loop:
`func name()`

If anything in loop, 
```
func name(
   param: val
)
```

The empty condition actually adds a line break.